<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SaveFileInfoWidget</class>
 <widget class="QWidget" name="SaveFileInfoWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>596</width>
    <height>657</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="areaNameWidget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Save Area Name</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="areaNameComboBox"/>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string>ID </string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="areaNameIDSpinBox">
        <property name="maximumSize">
         <size>
          <width>40</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="maximum">
         <number>65535</number>
        </property>
        <property name="value">
         <number>0</number>
        </property>
        <property name="displayIntegerBase">
         <number>16</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="realtimeWidget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Realtime at save</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="realtimeYearsSpinBox">
        <property name="suffix">
         <string> y</string>
        </property>
        <property name="minimum">
         <number>-1</number>
        </property>
        <property name="maximum">
         <number>2000000001</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="realtimeDaysSpinBox">
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="suffix">
         <string> d</string>
        </property>
        <property name="minimum">
         <number>-1</number>
        </property>
        <property name="maximum">
         <number>365</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="realtimeHoursSpinBox">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="suffix">
         <string> h</string>
        </property>
        <property name="minimum">
         <number>-1</number>
        </property>
        <property name="maximum">
         <number>24</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="realtimeMinutesSpinBox">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="suffix">
         <string> m</string>
        </property>
        <property name="minimum">
         <number>-1</number>
        </property>
        <property name="maximum">
         <number>60</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="realtimeSecondsSpinBox">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="suffix">
         <string> s</string>
        </property>
        <property name="minimum">
         <number>-1</number>
        </property>
        <property name="maximum">
         <number>60</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="realtimeMillisecondsSpinBox">
        <property name="minimumSize">
         <size>
          <width>70</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="suffix">
         <string> ms</string>
        </property>
        <property name="minimum">
         <number>-1</number>
        </property>
        <property name="maximum">
         <number>1000</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="realtimeMicrosecondsSpinBox">
        <property name="minimumSize">
         <size>
          <width>70</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="suffix">
         <string> Î¼s</string>
        </property>
        <property name="minimum">
         <number>-1</number>
        </property>
        <property name="maximum">
         <number>1000</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="playtimeWidget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Playtime</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="playtimeHSpinBox">
        <property name="suffix">
         <string> hr.</string>
        </property>
        <property name="minimum">
         <number>-1</number>
        </property>
        <property name="maximum">
         <number>2000000001</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="playtimeMSpinBox">
        <property name="suffix">
         <string> min.</string>
        </property>
        <property name="minimum">
         <number>-1</number>
        </property>
        <property name="maximum">
         <number>60</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="playtimeSSpinBox">
        <property name="suffix">
         <string> sec.</string>
        </property>
        <property name="minimum">
         <number>-1</number>
        </property>
        <property name="maximum">
         <number>60</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="playtimeMSDoubleSpinBox">
        <property name="suffix">
         <string> ms</string>
        </property>
        <property name="minimum">
         <double>-1.000000000000000</double>
        </property>
        <property name="maximum">
         <double>1000.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>16.666667000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Times Saved</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="timesSavedSpinBox">
        <property name="maximum">
         <number>65535</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="igTime" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>In Game Days</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="igDaysPassedSpinBox">
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximum">
         <number>2386</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Current Day</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_2" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>Night Starts at 70%, One Day/Night Cycle is 10 minutes</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="currentDayHorizontalSlider">
           <property name="minimum">
            <number>100</number>
           </property>
           <property name="maximum">
            <number>1799900</number>
           </property>
           <property name="singleStep">
            <number>37500</number>
           </property>
           <property name="value">
            <number>100</number>
           </property>
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="tickPosition">
            <enum>QSlider::TickPosition::TicksAbove</enum>
           </property>
           <property name="tickInterval">
            <number>0</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>Each transition skips through 30 seconds worth of time</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
